Jenkinsfile

```diff
@@ -457,7 +457,11 @@ pipeline {
             values '2.1',       // Legacy Apache CassandraⓇ
                    '3.0',       // Previous Apache CassandraⓇ
                    '3.11',      // Current Apache CassandraⓇ
-                   '4.0'        // Development Apache CassandraⓇ
+                   '4.0',       // Development Apache CassandraⓇ
+                   'dse-5.1',   // Legacy DataStax Enterprise
+                   'dse-6.0',   // Previous DataStax Enterprise
+                   'dse-6.7',   // Previous DataStax Enterprise
+                   'dse-6.8.0'  // Current DataStax Enterprise
```

Comment:
Around line 218-222, we list out the Cassandra versions that you can choose from for triggering a specific build. It would be more complete to add the DSE versions there as well.